assets/js/blocks/products/all-products/index.js

```diff
@@ -74,4 +73,37 @@ registerBlockType( 'woocommerce/all-products', {
 			</div>
 		);
 	},
+};
+
+/**
+ * Register and run the "All Products" block.
+ */
+registerBlockType( 'woocommerce/all-products', {
+	...blockSettings,
+	/**
+	 * Deprecation rule to handle the previous default rows which was 1 instead of 3.
+	 */
+	deprecated: [
+		{
+			attributes: Object.assign( blockSettings.attributes, {
+				rows: {
+					type: 'number',
+					default: 1,
+				},
+			} ),
+			save: blockSettings.save,
+		},
+	],
 } );
+
+addFilter(
+	'blocks.getBlockAttributes',
+	'woocommerce-blocks/get-block-attributes',
+	setBlockAttributeDefaults
+);
+
+addFilter(
+	'editor.BlockListBlock',
+	'woocommerce-blocks/with-default-attributes',
+	withDefaultAttributes
+);
```

Comment:
As discussed in slack, let's move these into their own location. Since this is an editor only thing, we can probably put this in `js/editor/filters.js` then in the main entry file (which is built to the `wc-blocks` handle) we can import it directly.  This will also require updating the `sideEffects` property in `package.json` to keep webpack from dropping the import during treeshaking.

Reply:
I've added to `assets/js/filters` and handled in sideEffects. You can see the latest code now.